mrp_production_estimated_cost/models/stock_quant.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# © 2016 Ainara Galdona - AvanzOSC
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from openerp import api, fields, models
+
+
+class StockQuant(models.Model):
```

Comment:
This shouldn't go here, because this is a stock feature, not an mrp stimated cost feature.